### PR TITLE
wip excluding sys files

### PR DESCRIPTION
**Pull request by karniwl**
- [ ] Ready for review
- [ ] Follows CONTRIBUTING rules
- [ ] Reviewed by Snyk internal team

### CODE REVIEW

File: lib/index.ts [hunk 1]
```diff
@@ -215,15 +215,20 @@ async function getManifestFiles(targetImage: string, options?: any) {
...
 
   const globs = options.manifestGlobs as string[];
   const docker = new Docker(targetImage, options);
 
-  let files = await docker.findGlobs(globs, excludeGlobs);
+  let files = await docker.findGlobs(globs, excludeGlobs, "/", false);
+
+  files.forEach(async (dir) => {
+    const nextFiles = await docker.findGlobs(globs, excludeGlobs, dir, true);
+    files.concat(nextFiles);
```
Comment (mladkau) [inline lib/index.ts]: this is a NOP: `The concat() method is used to merge two or more arrays. This method does not change the existing arrays, but instead returns a new array.` see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/concat

Comment (karniwl) [reply lib/index.ts]: thanks! silly me :)

File: lib/index.ts [hunk 2]
```diff
@@ -215,15 +216,30 @@ async function getManifestFiles(targetImage: string, options?: any) {
...
   }
 
   const globs = options.manifestGlobs as string[];
   const docker = new Docker(targetImage, options);
 
-  let files = await docker.findGlobs(globs, excludeGlobs);
+  const interFiles = await docker.findGlobs(globs, excludeGlobs, "/", false);
+
+  let files: string[] = flatten(
+    Promise.all(
```
Comment (mladkau) [inline lib/index.ts]: Nice one - this should reduce the overhead of running `ls` multiple times!

Comment (karniwl) [reply lib/index.ts]: but the test still fails :(

**Pull request closed without merging**